src/internal/direction.ts

```diff
@@ -15,6 +20,15 @@ export function getScrollInlineStart(element: HTMLElement) {
   return isRtl(element) ? Math.floor(element.scrollLeft) * -1 : Math.ceil(element.scrollLeft);
 }
 
+/**
+ * The clientX position needs to be converted so it is relative to the right of
+ * the document in order for computations to yield the same result in both
+ * element directions.
+ */
+export function getLogicalClientX(event: PointerEvent, IsRtl: boolean) {
```

Comment:
If you want to separate `isRtl` (a function) and this argument, you could call it `rtlState` or `currentRtl`.

P.S. renaming `isRtl` -> `getIsRtl` still makes sense, just not in this PR. Are you planning to do that still?

Reply:
I'm still planning to rename it in a separate PR.